conanfile.py

```diff
@@ -9,7 +9,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "6.6.25"
+    version = "6.6.26"
```

Comment:
Regardless of how we implement this; will require at least a MINOR version change to `6.7.0` to signify the added call that a downstream consumer can begin to use starting here.

Reply:
ack